libs/domain/cart/entries/src/entries.component.ts

```diff
@@ -1,169 +1,67 @@
-import { CartComponentMixin, CartEntry, CartService } from '@spryker-oryx/cart';
-import { resolve } from '@spryker-oryx/di';
-import { ContentController } from '@spryker-oryx/experience';
-import { ProductService } from '@spryker-oryx/product';
-import { asyncValue, hydratable } from '@spryker-oryx/utilities';
-import { html, TemplateResult } from 'lit';
+import { CartComponentMixin } from '@spryker-oryx/cart';
+import { ContentMixin, defaultOptions } from '@spryker-oryx/experience';
+import { hydratable, i18n } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
 import { repeat } from 'lit/directives/repeat.js';
-import { BehaviorSubject, combineLatest, map, of, switchMap, tap } from 'rxjs';
+import { RemoveByQuantity } from '../../entry/src';
```

Comment:
If it's cross package you can use alises `@spryker-oryx/...`

Reply:
but it's same package

Reply:
Assume you need `@spryker-oryx/cart/entry` while you in `@spryker-oryx/cart/entries`